modules/sql-engine/src/integrationTest/sql/types/time/time_parsing.test

```diff
@@ -22,8 +22,8 @@ SELECT '14:42:04.999'::TIME(3)::VARCHAR
 ----
 14:42:04.999
 
+# trailing zeros get truncated
 query I
-SELECT '14:42:04.999'::TIME(6)::VARCHAR
+SELECT '14:42:04.000000'::TIME(6)::VARCHAR
```

Comment:
i don`t understand here, you rewrite valid test into different one ?
i suppose you just need to append something like : SELECT '14:42:04.999000'::TIME(6)::VARCHAR, wdyt ?

Reply:
After Andrew comment I decided to copy original test from `time_parsing_ignore.test` :thinking:

Reply:
But in general I agree, I think the comment is enough, so I did as you suggested.